### ISSUE TITLE
Training loss not changing

### ISSUE THREAD

**Issue by uraibeef** [liuheng92/tensorflow_PSENet#87]
Hello, i try to re-train on my dataset in the same format like your training dataset, and when i training doesn't have an error but my model training loss not changing it's stay at 1.0303 every step.. What's wrong? and what should i do?

**Comment by wdh234** [comment]
训练数据集是否有模糊标注，即1477,609,1556,604,1558,633,1478,638,####

**Comment by lipeng1109** [comment]
我也有同样的问题，请求帮助

**Comment by uraibeef** [comment]
训练数据集是否有模糊标注，即1477,609,1556,604,1558,633,1478,638,####

ในวันที่ อ. 2 ก.พ. 2021 เวลา 10:02 lipeng1109 <notifications@github.com>
เขียนว่า:
